### PR TITLE
Enterprise events datetime format

### PR DESCRIPTION
**Pull request by carycheng77**
Ran BoxEventsManagerTestIntegration with new datetime format and passed tests and retrieved same results from previous datetime format

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @carycheng77 has signed the CLA. Thanks for the pull request!

File: Box.V2/Managers/BoxEventsManager.cs [hunk 1]
```diff
@@ -39,7 +39,9 @@ public BoxEventsManager(IBoxConfig config, IBoxService service, IBoxConverter co
                                                                         DateTime? createdAfter = null,
                                                                         DateTime? createdBefore = null)
         {
-            var createdAfterString = createdAfter.HasValue ? createdAfter.Value.ToString(Constants.RFC3339DateFormat) : null;
+            var createdAfterString = createdAfter.Value.ToString(createdAfter.Value.Kind == DateTimeKind.Utc
```
Comment (coolboy) [inline Box.V2/Managers/BoxEventsManager.cs]: Where are the integration test? What about the `o` format?

Comment (ohadschn) [reply Box.V2/Managers/BoxEventsManager.cs]: More importantly, what about the proper conversion to UTC ? https://msdn.microsoft.com/en-us/library/system.datetime.touniversaltime(v=vs.110).aspx

**Comment by ohadschn** [comment]
@carycheng77 @coolboy looks like you removed the `createdAfter.HasValue` check, this will result in `InvalidOperationException` if called with (the default) null value!

Also, why did you remove `WebUtility.UrlEncode`?

**Comment by carycheng77** [comment]
@ohadschn That was a mistake on my part! 

I submitted a new pr with the null check put back in place and the url encoding

#403

**Pull request merged**